manticore/native/cpu/abstractcpu.py

```diff
@@ -716,7 +719,7 @@ def _raw_read(self, where: int, size=1) -> bytes:
         elif isinstance(map, AnonMap):
             data = bytes(map._data[start : start + size])
         else:
-            data = b"".join(self.memory[where : where + size])
+            data = b"".join(self.memory.read(where, size, force=force))
```

Comment:
For future reference: `memory.__getitem__` is just a wrapper around `memory.read`, so this change was made for the sake of getting access to the `force` parameter.

Reply:
Yes, that is correct.